src/Options/index.js

```diff
@@ -520,3 +523,18 @@ export interface AuthAdapter {
   */
   enabled: ?boolean;
 }
+
+export interface LogLevelUses {
+  /* Log level used by the after hook trigger, default is 'info'.
```

Comment:
```suggestion
  /* Log level used by the Cloud Code Triggers `afterSave`, `afterDelete`, `afterSaveFile`, `afterDeleteFile`, `afterFind`, `afterLogout`. Default is `info`.
```

I think we should name each trigger individually to avoid confusion. There is the `afterLiveQueryEvent` but I'm not sure whether it is within the scope of this trigger log function for example.